Tests/Unit/InMemoryRefreshTokenManagerTest.php

```diff
@@ -21,47 +21,109 @@ public function testClearExpired(): void
     }
 
-    private function buildClearExpiredTestData(): array
+    private function buildTestData(callable $successFunction): array
     {
-        $validRefreshTokens = [
-            '1111' => $this->buildRefreshToken('1111', '+1 day'),
-            '2222' => $this->buildRefreshToken('2222', '+1 hour'),
-            '3333' => $this->buildRefreshToken('3333', '+1 second'),
-            '4444' => $this->buildRefreshToken('4444', 'now'),
+        $data = [
+            [
+                'identifier' => '1111',
+                'dateOffset' => '+1 day',
+                'revoked' => true,
+                'expired' => false,
+            ],
+            [
+                'identifier' => '2222',
+                'dateOffset' => '+1 hour',
+                'revoked' => false,
+                'expired' => false,
+            ],
+            [
+                'identifier' => '3333',
+                'dateOffset' => '+1 second',
+                'revoked' => true,
+                'expired' => false,
+            ],
+            [
+                'identifier' => '4444',
+                'dateOffset' => 'now',
+                'revoked' => false,
+                'expired' => false,
+            ],
+            [
+                'identifier' => '5555',
+                'dateOffset' => '-1 day',
+                'revoked' => true,
+                'expired' => true,
+            ],
+            [
+                'identifier' => '6666',
+                'dateOffset' => '-1 hour',
+                'revoked' => false,
+                'expired' => true,
+            ],
+            [
+                'identifier' => '7777',
+                'dateOffset' => '-1 second',
+                'revoked' => true,
+                'expired' => true,
+            ],
         ];
 
-        $expiredRefreshTokens = [
-            '5555' => $this->buildRefreshToken('5555', '-1 day'),
-            '6666' => $this->buildRefreshToken('6666', '-1 hour'),
-            '7777' => $this->buildRefreshToken('7777', '-1 second'),
-        ];
+        $response = [];
+        foreach ($data as $item) {
+            $identifier = $item['identifier'];
+            $refreshToken = $this->buildRefreshToken(
+                $identifier,
+                $item['dateOffset'],
+                $item['revoked']
+            );
+            $response['input'][$identifier] = $refreshToken;
 
-        return [
-            'input' => $validRefreshTokens + $expiredRefreshTokens,
-            'output' => $validRefreshTokens,
-        ];
+            if ($successFunction($item)) {
+                $response['output'][$identifier] = $refreshToken;
+            }
+        }
+
+        return $response;
     }
 
-    private function buildRefreshToken(string $identifier, string $modify): RefreshToken
+    private function buildRefreshToken(string $identifier, string $modify, bool $revoked): RefreshToken
```

Comment:
@franjo-zadelj-trikoder Another inconsistency. In some cases the $revoked argument has a default value `false`, here it doesn't. Please make that consistent.

Reply:
explained above